blocks/text.js

```diff
@@ -689,3 +689,90 @@ Blockly.Blocks['text_prompt'] = {
   mutationToDom: Blockly.Blocks['text_prompt_ext'].mutationToDom,
   domToMutation: Blockly.Blocks['text_prompt_ext'].domToMutation
 };
+
+Blockly.Blocks['text_count'] = {
+  /**
+   * Block for counting how many times one string appears within another string.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit({
+      'message0': Blockly.Msg.TEXT_COUNT_MESSAGE0,
+      'args0': [
+        {
+          'type': 'input_value',
+          'name': 'SUB',
+          'check': 'String',
+        },
+        {
+          'type': 'input_value',
+          'name': 'TEXT',
+          'check': 'String',
+        },
+      ],
+      'output': 'Number',
+      'inputsInline': true,
+      'colour': Blockly.Blocks.math.HUE,
+      'tooltip': Blockly.Msg.TEXT_COUNT_TOOLTIP,
+      'helpUrl': Blockly.Msg.TEXT_COUNT_HELPURL,
```

Comment:
There is a trailing comma on this line.  That's legal in the latest JS spec, but will break older browsers (meaning IE).  It's also illegal in JSON, meaning problems when this is used in Android and iOS.  Recommend removing this comma.  Here and other JSON blocks below.